.github/workflows/publish.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Python distributions to PyPI and TestPyPI
+on: push # Runs your workflow when you push a commit or tag.
+# on:
+#   release:
+#     types: [published] # triggered whenever a new GitHub release is published
```

Comment:
Can we remove redundant comments?

Reply:
Yes - sorry should have give more context here. Currently we're running on push to main. This commented out block only triggers the workflow on release (below)  We could only trigger the workflow to only run if if we use the Github release feature
<img width="1128" alt="Screen Shot 2022-12-23 at 7 42 30 AM" src="https://user-images.githubusercontent.com/19919899/209243488-8b037e6a-8856-4296-ae93-e20361b05c18.png">